src/ace/managers/ptplayer.c

```diff
@@ -1791,7 +1794,7 @@ static void mt_toneporta_nc(
 	tChannelStatus *pChannelData, volatile tChannelRegs *pChannelReg
 ) {
 	if(pChannelData->n_wantedperiod) {
-		UWORD uwNew;
+		WORD uwNew;
```

Comment:
since you've changed `UWORD` to `WORD`, the prefix should be changed from `uw` to `w`

Reply:
I can't seem to add suggestion for that change, too much lines :(